### PR TITLE
fix: [PIE-2730]: Fixed alignment issues of RunPipeline form

### PR DESCRIPTION
**Pull request by goyalsonali**
Summary:

1. Aligned New Service with respect to input
2. Aligned connector field of infrastructure
3. Aligned the icon in cluster details
4. Aligned connector field for all manifests and artifacts.

 Jira Links:

https://harness.atlassian.net/browse/PIE-2730

Screenshots:
Earlier:
Screenshot 2022-03-08 at 11 47 37 AM

Screenshot 2022-03-08 at 11 46 44 AM

Screenshot 2022-03-08 at 5 15 47 PM

Screenshot 2022-03-08 at 5 16 32 PM

Now:
Screenshot 2022-03-08 at 5 24 30 PM

Screenshot 2022-03-08 at 11 57 32 AM

Screenshot 2022-03-08 at 5 10 06 PM

Screenshot 2022-03-08 at 5 10 12 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by goyalsonali** [comment]
retrigger titlecheck

**Comment by goyalsonali** [comment]
retrigger coverage

**Comment by goyalsonali** [comment]
retrigger jest

**Comment by goyalsonali** [comment]
trigger rebase

**Comment by goyalsonali** [comment]
retrigger jest

**Comment by goyalsonali** [comment]
trigger cypress

File: src/modules/75-cd/components/PipelineSteps/GcpInfrastructureSpec/GcpInfrastructureSpec.tsx [hunk 1]
```diff
@@ -252,6 +252,7 @@ const GcpInfrastructureSpecEditable: React.FC<GcpInfrastructureSpecEditableProps
                 />
                 {getMultiTypeFromValue(formik.values.connectorRef) === MultiTypeInputType.RUNTIME && !readonly && (
                   <ConfigureOptions
+                    style={{ marginTop: `22px ` }}
```
Comment (chiragp1212) [inline src/modules/75-cd/components/PipelineSteps/GcpInfrastructureSpec/GcpInfrastructureSpec.tsx:255]: add this via a class and not inline style

Comment (goyalsonali) [reply src/modules/75-cd/components/PipelineSteps/GcpInfrastructureSpec/GcpInfrastructureSpec.tsx:255]: Okay

Comment (AshwinBhatkal-Harness) [reply src/modules/75-cd/components/PipelineSteps/GcpInfrastructureSpec/GcpInfrastructureSpec.tsx:255]: @chiragp1212 This is under a layout.horizontal. Wouldn't align-items: "center" be a better option?

Comment (AshwinBhatkal-Harness) [reply src/modules/75-cd/components/PipelineSteps/GcpInfrastructureSpec/GcpInfrastructureSpec.tsx:255]: @goyalsonali Did you try using align-items:center?

**Review by bhavya-sinha — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**